test/integration/controller_test.go

```diff
@@ -1422,6 +1102,11 @@ func TestBasicFlowsAsyncBinding(t *testing.T) {
 		t.Fatalf("error creating Instance: %v", err)
 	}
 
+	// Polling is going to take at least 3 seconds, with a 1-second break between the first poll and the second
+	// and a 2-second break between the second poll and the third. Let's sleep here so that we don't risk
+	// timing out while waiting for the instance condition to be ready in the following wait.
+	time.Sleep(3 * time.Second)
```

Comment:
What would time out, and why can't the `WaitForInstanceCondition` handle the waiting?

Reply:
Duh, don't know why that concept escaped me. I am working on more changes to integration tests. Would it be OK if I change this in a PR in the near future?